### ISSUE TITLE
Hlin(n)a misspeling

### ISSUE THREAD

**Issue by trotsiuk** [remoteforests/remoteforests_disturbance#4]
Ahoj @ondrej-vostarek I just found out that there is inconsistency in `Hlinna` stand in its name. Please have a look shall it be possible to correct if you will have time

![screen shot 2018-07-26 at 15 53 12](https://user-images.githubusercontent.com/15097779/43266696-52f46b90-90ec-11e8-95d5-00619b9b4a1e.png)

**Comment by ondrej-vostarek** [comment]
Ahoj @trotsiuk , 

hope you are doing well!

Problem solved - I have united the stand name as "Hlina", also there may be some changes in the names of stands and locations in the future to have them united between the web and the database, but this is up to a discussion after the holidays.

I would like to ask you if you hadn't time to have a look on the function to calculate the correlation between the core and the master chronology we were talking about last time? 

Thank you and have a nice day,
Ondra.

**Comment by trotsiuk** [comment]
Ahoj @ondrej-vostarek thank you for solving.

Sorry I don't fully remember about the correlation. Do we have an example script for it?

**Comment by ondrej-vostarek** [comment]
Unfortunately we did not. The aim was to mimic the function from ARSTAN to be able to calculate the correlation of a core with the master chronology in R, so we could obtain this information for cores that are already in the database. I have tried to do this via dplR function corr.rwl.seg but I failed to obtain similar values as were obtained from the ARSTAN (attached an example of correlation calculated by Milos in ARSTAN). 
[correlation.xlsx](https://github.com/trotsiuk/remoteforests_disturbance/files/2241687/correlation.xlsx)
